lang/de.json

```diff
@@ -1056,6 +1057,8 @@
     "Show parent": "Übergeordneten anzeigen",
     "Show unassigned payments": "Nicht zugeordnete Zahlungen anzeigen",
     "Showing": "Zeige",
+    "Signature saved": "Unterschrift gespeichert",
+    "Signature Upload Failed": "Upload fehlgeschlagen",
```

Comment:
Rename occurrences from Signature Upload Failed to Upload failed and remove this translation